README.md

```diff
@@ -7,4 +7,4 @@ A set of [Gradle](http://gradle.org) plugins to build and use [Xtext](http://xte
 
 The documentation is hosted on the [project's website](http://xtext.github.io/xtext-gradle-plugin/).
 
-To build and install locally run `./gradlew install -Dorg.gradle.project.version=<version>` e.g. with `<version>` set to `1.0.22-SNAPSHOT`.
+To build and install locally run ` ./gradlew clean install -Dorg.gradle.project.version=<version> -Dmaven.repo.local=.m2` e.g. with `<version>` set to `1.0.22-SNAPSHOT`.
```

Comment:
Can you revert this please? You don't need to run clean with Gradle and the custom m2 will break the tests.

Reply:
checked